src/autoPagination.ts

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const makeRequest = require('./makeRequest');
```

Comment:
sometimes we use `import makeRequest = require` and sometimes `const makeRequest = require` is it intentional?

Reply:
(Copied from offline discussion)

This was intentional to avoid emitting the `Object.defineProperty(exports, "__esModule", { value: true });` line.